checkov/common/util/decorators.py

```diff
@@ -22,7 +22,10 @@ def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
         end = default_timer()
 
         func_path = f"{func.__code__.co_filename.replace('.py', '')}.{func.__name__}"
-        logging.info(f"'{func_path}' took: {timedelta(seconds=end - start)}")
+        info = f"'{func_path}' took: {timedelta(seconds=end - start)}\n"
+        with open('time_it.txt', 'a') as f:
```

Comment:
how will we see it in the cloud ?

Reply:
actually, this is for local runs :) if we want it in s3 and saved for later - it's not from here.
I can remove it, but personally, it helped me track runs and not lose the latest run accidently.

Reply:
so maybe somehow we can do it file for local and console out for cloud ?

Reply:
I am doing both :) I didnt removed the previous code

Reply:
oh cool, missed that